facebook/OpenBIC
# wc: mb: Modify 2nd source vr address

Summary:
- Modify one specific 2nd vr device's slave address.

Test Plan:
- Build Code: PASS
- Check sensor read from BIC console: PASS

Log:
- BIC console:
  ```
  uart:~$ platform sensor list_all
  ---------------------------------------------------------------------------------
  [0x1 ] MB Inlet Temp            : tmp75      | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 30.0
  [0x2 ] MB Outlet Temp           : tmp75      | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 34.0
  [0x3 ] IOM Temp                 : tmp75      | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 31.0
  [0x8 ] SSD0 Temp                : nvme       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 33.0
  [0x5 ] SOC CPU Temp             : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 38.0
  [0x6 ] SOC Therm Margin         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | -62.0
  [0x7 ] CPU TjMax                : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 100.0
  [0xa ] DIMMA Temp               : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 33.0
  [0xb ] DIMMC Temp               : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 30.0
  [0xc ] DIMME Temp               : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 33.0
  [0xd ] DIMMG Temp               : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 32.0
  [0x60] CPU Pwr                  : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 56.0
  [0x20] P12V_STBY Vol            : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 12.31
  [0x26] P12V_DIMM Vol            : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 12.64
  [0x22] P3V3_STBY Vol            : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 3.344
  [0x24] P1V05_PCH Vol            : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.44
  [0x21] P3V_BAT Vol              : adc        | access[O] | poll[O] 3600 sec | 4byte_acur_read_success   | 3.105
  [0x28] P3V3_M2 Vol              : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 3.334
  [0x25] P5V_STBY Vol             : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 5.23
  [0x23] P1V8_STBY Vol            : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.816
  [0x27] P1V2_STBY Vol            : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.196
  [0x2d] VCCD VR Vol              : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.144
  [0x2e] FAON VR Vol              : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.56
  [0x2c] EHV VR Vol               : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.799
  [0x2a] VCCIN VR Vol             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.783
  [0x2b] FIVRA VR Vol             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.798
  [0x44] VCCD VR Cur              : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.417
  [0x45] VCCD VR Cur in           : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.529
  [0x46] FAON VR Cur              : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 8.296
  [0x43] EHV VR Cur               : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.476
  [0x41] VCCIN VR Cur             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 20.437
  [0x42] FIVRA VR Cur             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 4.367
  [0x11] VCCD SPS Temp            : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 40.0
  [0x12] FAON SPS Temp            : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 39.625
  [0x10] EHV SPS Temp             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 38.875
  [0xe ] VCCIN SPS Temp           : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 42.625
  [0xf ] FIVRA SPS Temp           : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 39.812
  [0x69] VCCD VR Pout             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 2.7
  [0x6a] FAON VR Pout             : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 8.843
  [0x68] EHV VR Pout              : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.812
  [0x66] VCCIN VR Pout            : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 36.250
  [0x67] FIVRA VR Pout            : tps53689   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 7.585
  [0x4 ] PCH Temp                 : pch        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 47.0
  [0x62] DIMMA PMIC_Pout          : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.875
  [0x63] DIMMC PMIC_Pout          : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.750
  [0x64] DIMME PMIC_Pout          : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.750
  [0x65] DIMMG PMIC_Pout          : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.750
  [0x6b] IOM INA Pwr              : ina230     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 18.0
  [0x47] IOM INA Cur              : ina230     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.500
  [0x2f] IOM INA vol              : ina230     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 11.995
  [0x9 ] HSC Temp                 : mp5990     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 32.0
  [0x29] HSC Input Vol            : mp5990     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 11.937
  [0x40] HSC Output Cur           : mp5990     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 6.0
  [0x61] HSC Input Pwr            : mp5990     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 72.0
  ---------------------------------------------------------------------------------
  ```
